### PR TITLE
add language changer to all README languages

### PR DESCRIPTION
**Pull request by DedBash**
I thought you could also put the language changer in the respective README files so that it is easier to navigate through if you have clicked the wrong way.
i linked the main README with the US flagger and removed the own flage in the respective language README files so you don't see it there unnecessarily
If this is not wanted you can simply close the request

### CODE REVIEW

File: translations/README_vi_VN.md [hunk 1]
```diff
@@ -15,6 +15,19 @@
 </p>
 
 
+# Translations
```
Comment (sant1ago-da-hanoi) [inline translations/README_vi_VN.md:18]: # Đọc bằng ngôn ngữ khác

Comment (sant1ago-da-hanoi) [inline translations/README_vi_VN.md:18]: "Translations" => "Đọc bằng ngôn ngữ khác"

Comment (DedBash) [reply translations/README_vi_VN.md:18]: I thought about leaving this in english for the reason that if you accidentally come to the language you should still read this I would be willing to put it in the national languages in brackets behind it but in general I had it just like the disclaimer thought to just leave this in english

**Comment by PencilNavigator** [comment]
Unnesscery. There is a "back" button on your browser for you to easily go back to the english README to choose the correct language if you want.

Before there was a language chooser in all Translation README's but it was removed due to inconsistency for maintenence.

**Pull request closed without merging**